framework/db/QueryBuilder.php

```diff
@@ -1094,7 +1094,7 @@ public function buildInCondition($operator, $operands, &$params)
                 foreach ($value->params as $n => $v) {
                     $params[$n] = $v;
                 }
-            } else {
+            } elseif (gettype($value) !== 'integer' && gettype($value) !== 'double') {
```

Comment:
=>

```php
elseif (!is_int($value) && !is_float($value)) {
```

`gettype()` make sence only when you need result as string to compare against some var or to use inside exceptions.